Userland/Applications/MailSettings/MailSettingsWindow.gml

```diff
@@ -0,0 +1,145 @@
+@GUI::Frame {
+    fill_with_background_color: true
+
+    layout: @GUI::VerticalBoxLayout {
+        margins: [10, 10, 10, 10]
+        spacing: 5
+    }
+
+    @GUI::GroupBox {
+        title: "Server Settings"
+        fixed_height: 170
+
+        layout: @GUI::VerticalBoxLayout {
+            margins: [8, 16, 8, 8]
+            spacing: 2
+        }
+
+        @GUI::Widget {
+            layout: @GUI::HorizontalBoxLayout {
+                spacing: 16
+            }
+
+            @GUI::Label {
+                fixed_width: 32
+                fixed_height: 32
+                name: "server_settings_image_label"
+            }
+
+            @GUI::Label {
+                text: "Mail server connectivity settings"
```

Comment:
This feels like placeholder text. How about something more user friendly, like "These settings let you pick which server you would like to fetch your mail from."